qml/FeedsPage.qml

```diff
@@ -258,6 +258,8 @@ BasePage {
                     }
                     onClicked:
                     {
+                        //revert check
+                        checked = Qt.binding( function () { return configEnabled; } );
```

Comment:
I didn't realize check was actually a property of the Switch Qt component... It's a bit weird to rewrite some of their properties. Couldn't we do something more "classic" like
"onConfigEnabledChanged: checked = configEnabled" ?